ZelBack/src/services/benchmarkService.js

```diff
@@ -139,6 +139,14 @@ async function getInfo(req, res) {
   return res ? res.json(response) : response;
 }
 
+async function getBenchPublicIp(req, res) {
```

Comment:
please rename to getPublicIp so it is the same name as the rpc method and different function name to the function in fluxCommunicaation

Reply:
Adjusted.